crates/matrix-sdk/src/widget/machine/from_widget.rs

```diff
@@ -245,10 +254,23 @@ impl From<update_delayed_event::unstable::Response> for UpdateDelayedEventRespon
     }
 }
 
-/// The response to the widget that it received the to-device event.
-/// Only used as the response for the successful send case.
-/// FromWidgetErrorResponse will be used otherwise.
-/// This is intentionally an empty tuple struct (not a unit struct), so that it
-/// serializes to `{}` instead of `Null` when returned to the widget as json.
-#[derive(Serialize, Debug)]
-pub(crate) struct SendToDeviceEventResponse {}
+/// Send to device request response.
```

Comment:
This comment is much terser, could we please bring back some information the comment contained?